src/app/utils/formatUrlWithQuery.ts

```diff
@@ -4,7 +4,7 @@ export type QueryParams = {
 
 export const formatUrlWithQuery = (queryParams: QueryParams, path: string) => {
 	const queryString = Object.entries(queryParams)
-		.filter(([key, value]) => value)
+		.filter(([value]) => value)
```

Comment:
Hum.... Are we sure of this ?

Reply:
we don't need the `[ ]`around value but except that it looks good. What's bothering you ?